pprojection/main.go

```diff
@@ -0,0 +1,112 @@
+package main
```

Comment:
delete before merge, this is just a convenient test harness to be able to iterate quickly on ensuring the resulting jpeg looks the way we want